utils/docker/examples/README.md

```diff
@@ -0,0 +1,547 @@
+# DAOS in Docker
+
+This document describes different ways to build and deploy Docker images for running application
+using a DAOS storage system.
```

Comment:
Client, server or both?

Reply:
With this PR, both use case are covered :)

- [ ] Improve README intro

Reply:
fixed with commit [7a87891](https://github.com/daos-stack/daos/pull/13076/commits/7a878910e3aeb241265270d8c66ae5c231c9de00)